unitary/alpha/quantum_world_test.py

```diff
@@ -345,9 +346,15 @@ def test_copy(simulator, compile_to_qubits):
     assert board.circuit is not board2.circuit
     assert board.effect_history == board2.effect_history
     assert board.effect_history is not board2.effect_history
+    assert board.effect_history_length == board2.effect_history_length
+    assert board.qubit_remapping_dict_length == board2.qubit_remapping_dict_length
```

Comment:
ideally these asserts should also have some sort a message (second argument to assert) to help callers debug issues

Reply:
I think most cases here are kind of self explanatory and would be a bit redundant to add messages. Thanks.